src/Magnum/Platform/EmscriptenApplication.cpp

```diff
@@ -562,8 +566,12 @@ void EmscriptenApplication::setupCallbacks(bool resizable) {
 
         if(element === document) return 1; /* EMSCRIPTEN_EVENT_TARGET_DOCUMENT */
         if(element === window) return 2; /* EMSCRIPTEN_EVENT_TARGET_WINDOW */
-        if('id' in element)
-            return allocate(intArrayFromString(element.id), 'i8', ALLOC_NORMAL);
+        if('id' in element) {
+            const bytes = lengthBytesUTF8(element.id) + 1;
+            const memory = _malloc(bytes);
+            stringToUTF8(element.id, memory, bytes);
+            return memory;
+        }
```

Comment:
Did you see any significant change in build output size with this change present? I hope malloc doesn't drag along tens of kilobytes of JS code that wouldn't be needed with plain allocate :)

Reply:
136kb before, 137kb after, 191kb with `ASSERTIONS` and `SAFE_HEAP`

Reply:
Nice. That's quite okay.